akka-distributed-data/src/main/scala/akka/cluster/ddata/protobuf/ReplicatedDataSerializer.scala

```diff
@@ -8,27 +8,27 @@ import java.{ util, lang => jl }
 import java.util.ArrayList
 import java.util.Collections
 import java.util.Comparator
-import java.util.TreeSet
 
 import scala.annotation.tailrec
 import akka.util.ccompat.JavaConverters._
-import scala.collection.immutable
 
+import scala.collection.immutable
```

Comment:
changes to the replicated data serializer as it had generic methods for `GeneratedMessage` which is now `GeneratedMessageV3`